testing/py3-subunit/APKBUILD

```diff
@@ -0,0 +1,25 @@
+# Contributor: Robert Sacks <robert@sacks.email>
+# Maintainer: Robert Sacks <robert@sacks.email>
+pkgname="py3-subunit"
+_pyname="python-subunit"
+pkgver="1.3.0"
+pkgrel=0
+pkgdesc="Python implementation of subunit test streaming protocol"
+url="https://launchpad.net/subunit"
+arch="noarch"
+license="Apache-2.0 BSD-3-Clause"
```

Comment:
From the source: https://bazaar.launchpad.net/~subunit/subunit/master/view/head:/python/subunit/_to_disk.py

> Licensed under either the Apache License, Version 2.0 or the BSD 3-clause license at the users choice

(make it Apache-2.0 OR BSD-3-Clause)

Reply:
Fixed